### PR TITLE
search result mobile view change

### PR DESCRIPTION
**Pull request by E-fais**
closes issue #174 
mobile view UI change.

### CODE REVIEW

**Comment by E-fais** [comment]
Abbreve - Google Chrome 26-11-2022 9 30 59 PM

**Comment by Njong392** [comment]
Looks great! I tried to fix a merge conflicts and broke one of the vercel deployment checks. I'll check it tomorrow. This is an amazing job you did though 💜 thank you!

**Comment by Njong392** [comment]
Okay, I fixed the failing check, but there is one more thing. There are some changes that were automatically made to the `package-lock.json` file and I would like for you to revert those.

You can check [here](https://gist.github.com/kumarsonsoff3/fbc6ecde519250a9f0a4d7ccc9993a1a) for why this happened, and how you can easily fix it.

**Comment by Njong392** [comment]
Noticing just now after fixing the bad check that your changes don't reflect on my browser tools. Perhaps you could sync your fork, pull recent changes from this main upstream, and see if anything went wrong.

Screenshot from 2022-11-29 19-48-23

**Comment by E-fais** [comment]
okay , let me check.

**Comment by Njong392** [comment]
Hello @E-fais any updates on this?

File: src/components/form.jsx [hunk 1]
```diff
@@ -108,20 +108,21 @@ const Form = () => {
...
               />
             </div>
-
-            <button
-              onClick={fetchData}
-              disabled={isLoading || !hasUserInputChanged}
-              className="bg-deeppurple text-ash font-bold rounded-xl hover:scale-110 p-2 mt-2 md:mt-0">
-              Search
-            </button>
+            <div className="flex gap-2 md:lg:flex-none ">
```
Comment (Njong392) [inline src/components/form.jsx:111]: I see that you added a `flex gap-2 md:lg:flex-none` here. The flex is applied only to the button, maybe that's why the change isn't working for me (?) The flex has to be somehow applied to the button and the search result.

Did the changes get applied to your browser though? It'll be cool if you add screenshots to the pull request.

**Review by Njong392 — CHANGES_REQUESTED** [review]

**Comment by E-fais** [comment]
Hey, I am really sorry. I was busy, i could not follow up...

**Comment by Njong392** [comment]
I think after seeing and merging #219 we can close this Pull request. What do u think?

**Comment by E-fais** [comment]
Yes, let's close this. Sorry for not responding.

**Pull request closed without merging**